src/engine/strat_engine/filesystem.rs

```diff
@@ -109,6 +156,29 @@ impl Recordable<FilesystemSave> for StratFilesystem {
     }
 }
 
+/// Return total bytes, total bytes used
```

Comment:
Consider returning total bytes and total bytes free. Let the caller decide how to use those.